collector/lib/NetworkConnectionInfoServiceComm.h

```diff
@@ -28,7 +28,7 @@ class INetworkConnectionInfoServiceComm {
 
   virtual sensor::NetworkConnectionInfoService::StubInterface* GetStub() = 0;
 
-  virtual std::unique_ptr<IDuplexClientWriter<sensor::NetworkConnectionInfoMessage>> PushNetworkConnectionInfoOpenStream(std::function<void(const sensor::NetworkFlowsControlMessage*)> receive_func) = 0;
+  virtual std::unique_ptr<IDuplexClientWriter<sensor::NetworkConnectionInfoMessage>> PushNetworkConnectionInfoOpenStream(std::function<void(const sensor::MsgToCollector*)> receive_func) = 0;
```

Comment:
Hm, I think it is dangerous to change the existing message, this will break installations which are using Collector and Sensor in different versions.
Is this something we have done before like this? What was the expierence? (e.g. support cases)

Preferably we find a solution which is backward compatible.

Reply:
As Sensor and Collector are deployed using the same Daemonset, we usually consider that a version mismatch can only be temporary. Introducing a new method here will make Sensor reply with a "method not found" if there is any mismatch.

Reply:
> As Sensor and Collector are deployed using the same Daemonset

Are they? Sensor has its own Deployment.

Reply:
@erthalion I think @ovalenti  refers to the same Secured Cluster deployment, in it they share the same version. I think its better to continue the discussion in the slack thread.